src/m365/spe/commands/containertype/containertype-add.spec.ts

```diff
@@ -236,3 +236,4 @@ describe(commands.CONTAINERTYPE_ADD, () => {
       new CommandError('Invalid request'));
   });
 });
+
```

Comment:
Why add an additional line-break here?

Reply:
Probably the auto code formatter 🤷
A test in this file was originally changed, but due to a `v11` breaking change, the test was removed.